src/externals/pio2/src/clib/ncparser.pl

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/perl 
+use strict;
+
+my $netcdf_incdir = $ARGV[0];
+my $pnetcdf_incdir = $ARGV[1];
+
```

Comment:
Also, there is only one change made in pio2 directory (cime/src/externals/pio2) in the ACME side that we need to retain. The change was to rename the XML man pages (./pio2/examples/basic/perl5lib/XML/man3/XML_Lite_Element.3 , ./pio2/examples/basic/perl5lib/XML/man3/XML_Lite.3) to remove colons.

(The old filenames were ./pio2/examples/basic/perl5lib/XML/man3/XML::Lite.3 , ./pio2/examples/basic/perl5lib/XML/man3/XML::Lite::Element.3)